docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+
+The exact nature of this mapping depends on how the Vertiq module is configured. Specifically it depends on how the *FC 2D/3D Mode* and the *Direction* parameters 
+are configured in the IQ Control Center. The *FC 2D/3D Mode* parameter can be set to either *2D* or *3D*. The *Direction* parameter can be set to *3D Clockwise*, 
+*3D Counterclockwise*, *2D Clockwise*, or *2D Counterclockwise*. The images below show these parameters in the IQ Control Center.
+
+.. figure:: ../_static/manual_images/throttle/throttle_fc_mode.png
+    :align: center
+    :width: 60%
+    :alt: FC 2D/3D Mode Parameter
+
+    FC 2D/3D Mode Parameter in IQ Control Center
+
+.. figure:: ../_static/manual_images/throttle/throttle_motor_direction.png
+    :align: center
+    :width: 60%
+    :alt: Motor Direction Parameter
+
+    Motor Direction Parameter in IQ Control Center
+
+For the mapping, all that matters for the *Direction* is whether it is 2D or 3D, the clockwise or counterclockwise selection only affects what direction a positive 
+throttle command means as covered in the :ref:`throttle_direction` section. This means that there are four meaningful combinations of these parameters that will result in 
+different mappings. Each of these combinations and the mapping they result in is covered below.
+
+.. _throttle_2d_2d_mapping:
+
+FC Mode = 2D, Motor Direction = 2D
+************************************
+This is the most commonly used configuration. With this setup, raw values map directly to throttle commands, meaning that raw values cannot map to negative throttle commands 
+and therefore cannot reverse the module. Raw values will only make the module spin in the module’s configured direction.
+
+More explicitly, the simple equation below details how to convert from a raw value to a throttle command percentage with this setup.
+
+.. figure:: ../_static/manual_images/throttle/throttle_2d_2d_equation.png
+    :align: center
+    :width: 40%
+    :alt: FC Mode = 2D, Motor Direction = 2D Throttle Mapping
+
+    Raw Value to Throttle Command Mapping Equation (FC Mode = 2D, Motor Direction = 2D)
+
+For example, a raw value of 0.5 will map to a throttle command of 50%, a raw value of 0.0 will map to a throttle command of 0%, and a raw value of 1.0 will map to a throttle 
+command of 100%. 
+
+The graph below illustrates this mapping. The X-axis is a raw value, and the Y-axis is the corresponding throttle command percentage.
+
+.. figure:: ../_static/manual_images/throttle/throttle_2d_2d_graph.png
+    :align: center
+    :width: 60%
+    :alt: FC Mode = 2D, Motor Direction = 2D Throttle Mapping  Graph
+
+    Raw Value to Throttle Command Mapping Graph (FC Mode = 2D, Motor Direction = 2D)
+
+FC Mode = 3D, Motor Direction = 2D
+************************************
+This configuration produces exactly the same results as the :ref:`throttle_2d_2d_mapping` setup, and so should generally be avoided in favor of that configuration.

```

Comment:
This configuration produces identical results to <mapping link here>, and should generally be avoided in favor of that configuration.